src/index.html

```diff
@@ -1,17 +1,184 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+  <!-- Первая часть -->
+
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      class="distance"
+      onsubmit="onSubmit()"
+      method="POST">
```

Comment:
Use correct new lines and indentations. Closing bracket should be on the same level as opening one.
Need to fix it for all your document.
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      class="distance"
      onsubmit="onSubmit()"
      method="POST"
    >
```